### PR TITLE
Clarify guidance for using Kestrel

### PR DESCRIPTION
**Pull request by danroth27**
Eventually we will have a complete discussion of Kestrel and ASP.NET hosting (#929), but adding a warning to the Servers doc for now.

### CODE REVIEW

File: aspnet/fundamentals/servers.rst [hunk 1]
```diff
@@ -167,12 +167,12 @@ Kestrel is a cross-platform web server based on `libuv <https://github.com/libuv
 
 Learn more about working with Kestrel to create :doc:`/tutorials/your-first-mac-aspnet`.
 
+.. note:: Kestrel is not hardened for internet facing traffic and should always be used behind a secure proxy server (ex. IIS, Nginx).
```
Comment (danroth27) [inline aspnet/fundamentals/servers.rst]: @blowdart @DamianEdwards  You guys ok with this language?

**Comment by danroth27** [comment]
Adding some minor doc cleanup as well.

@rachelappel @Rick-Anderson Please review

File: aspnet/fundamentals/servers.rst (hunk 1 above)
Comment (Rick-Anderson) [inline aspnet/fundamentals/servers.rst]: we don't use ex. replace with For Example

How about:
Kestrel on it's own, has vulnerabilities as an internet web server.  To secure Kestrel, it must be used behind a secure proxy server such as IIS or Nginx.

Comment (blowdart) [inline aspnet/fundamentals/servers.rst]: Rather harsh, How about "Kestrel is designed to be run behind a proxy (For example IIS or nginx) and should not be deployed directly facing the internet."

**Comment by Rick-Anderson** [comment]
:shipit: with blowdarts verbage.

**Pull request merged**